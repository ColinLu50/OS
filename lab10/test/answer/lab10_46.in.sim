9767
641
647
405
641
399
