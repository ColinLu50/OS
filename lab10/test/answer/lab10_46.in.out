FCFS
232 -> 352 -> 370 -> 226 -> 318 -> 253 -> 175 -> 368 -> 217 -> 49 -> 124 -> 105 -> 44 -> 293 -> 284 -> 171 -> 139 -> 100 -> 342 -> 107 -> 124 -> 101 -> 6 -> 271 -> 95 -> 121 -> 224 -> 201 -> 75 -> 284 -> 397 -> 324 -> 51 -> 28 -> 404 -> 273 -> 302 -> 22 -> 214 -> 231 -> 388 -> 298 -> 373 -> 107 -> 153 -> 256 -> 337 -> 159 -> 369 -> 27 -> 29 -> 252 -> 306 -> 206 -> 159 -> 245 -> 341 -> 265 -> 263 -> 224 -> 61 -> 365 -> 341 -> 170 -> 317 -> 23 -> 26 -> 312 -> 3 -> 272 -> 279 -> 276 -> 76 -> 88 -> 189 -> 397 -> 415 -> 333 -> 257 -> 146 -> 179
9767
SSTF
232 -> 231 -> 226 -> 224 -> 224 -> 217 -> 214 -> 206 -> 201 -> 189 -> 179 -> 175 -> 171 -> 170 -> 159 -> 159 -> 153 -> 146 -> 139 -> 124 -> 124 -> 121 -> 107 -> 107 -> 105 -> 101 -> 100 -> 95 -> 88 -> 76 -> 75 -> 61 -> 51 -> 49 -> 44 -> 29 -> 28 -> 27 -> 26 -> 23 -> 22 -> 6 -> 3 -> 245 -> 252 -> 253 -> 256 -> 257 -> 263 -> 265 -> 271 -> 272 -> 273 -> 276 -> 279 -> 284 -> 284 -> 293 -> 298 -> 302 -> 306 -> 312 -> 317 -> 318 -> 324 -> 333 -> 337 -> 341 -> 341 -> 342 -> 352 -> 365 -> 368 -> 369 -> 370 -> 373 -> 388 -> 397 -> 397 -> 404 -> 415
641
SCAN
232 -> 231 -> 226 -> 224 -> 217 -> 214 -> 206 -> 201 -> 189 -> 179 -> 175 -> 171 -> 170 -> 159 -> 153 -> 146 -> 139 -> 124 -> 121 -> 107 -> 105 -> 101 -> 100 -> 95 -> 88 -> 76 -> 75 -> 61 -> 51 -> 49 -> 44 -> 29 -> 28 -> 27 -> 26 -> 23 -> 22 -> 6 -> 3 -> 0 -> 245 -> 252 -> 253 -> 256 -> 257 -> 263 -> 265 -> 271 -> 272 -> 273 -> 276 -> 279 -> 284 -> 293 -> 298 -> 302 -> 306 -> 312 -> 317 -> 318 -> 324 -> 333 -> 337 -> 341 -> 342 -> 352 -> 365 -> 368 -> 369 -> 370 -> 373 -> 388 -> 397 -> 404 -> 415
647
C-SCAN
232 -> 231 -> 226 -> 224 -> 217 -> 214 -> 206 -> 201 -> 189 -> 179 -> 175 -> 171 -> 170 -> 159 -> 153 -> 146 -> 139 -> 124 -> 121 -> 107 -> 105 -> 101 -> 100 -> 95 -> 88 -> 76 -> 75 -> 61 -> 51 -> 49 -> 44 -> 29 -> 28 -> 27 -> 26 -> 23 -> 22 -> 6 -> 3 -> 0 -> 418 -> 415 -> 404 -> 397 -> 388 -> 373 -> 370 -> 369 -> 368 -> 365 -> 352 -> 342 -> 341 -> 337 -> 333 -> 324 -> 318 -> 317 -> 312 -> 306 -> 302 -> 298 -> 293 -> 284 -> 279 -> 276 -> 273 -> 272 -> 271 -> 265 -> 263 -> 257 -> 256 -> 253 -> 252 -> 245
405
LOOK
232 -> 231 -> 226 -> 224 -> 217 -> 214 -> 206 -> 201 -> 189 -> 179 -> 175 -> 171 -> 170 -> 159 -> 153 -> 146 -> 139 -> 124 -> 121 -> 107 -> 105 -> 101 -> 100 -> 95 -> 88 -> 76 -> 75 -> 61 -> 51 -> 49 -> 44 -> 29 -> 28 -> 27 -> 26 -> 23 -> 22 -> 6 -> 3 -> 245 -> 252 -> 253 -> 256 -> 257 -> 263 -> 265 -> 271 -> 272 -> 273 -> 276 -> 279 -> 284 -> 293 -> 298 -> 302 -> 306 -> 312 -> 317 -> 318 -> 324 -> 333 -> 337 -> 341 -> 342 -> 352 -> 365 -> 368 -> 369 -> 370 -> 373 -> 388 -> 397 -> 404 -> 415
641
C-LOOK
232 -> 231 -> 226 -> 224 -> 217 -> 214 -> 206 -> 201 -> 189 -> 179 -> 175 -> 171 -> 170 -> 159 -> 153 -> 146 -> 139 -> 124 -> 121 -> 107 -> 105 -> 101 -> 100 -> 95 -> 88 -> 76 -> 75 -> 61 -> 51 -> 49 -> 44 -> 29 -> 28 -> 27 -> 26 -> 23 -> 22 -> 6 -> 3 -> 415 -> 404 -> 397 -> 388 -> 373 -> 370 -> 369 -> 368 -> 365 -> 352 -> 342 -> 341 -> 337 -> 333 -> 324 -> 318 -> 317 -> 312 -> 306 -> 302 -> 298 -> 293 -> 284 -> 279 -> 276 -> 273 -> 272 -> 271 -> 265 -> 263 -> 257 -> 256 -> 253 -> 252 -> 245
399
